main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -240,6 +240,11 @@ object KindedAst {
 
     case class Tuple(elms: List[Pattern], loc: SourceLocation) extends Pattern
 
+    case class Record(pats: List[Record.RecordFieldPattern], pat: Option[Pattern], tvar: ast.Type.Var, loc: SourceLocation) extends KindedAst.Pattern
```

Comment:
Should `pat` be renamed to `extension` or `tail`? It denotes the `{ ... | r }` part.

Reply:
I think it is fine